drivers/sensor/lis2dw12/lis2dw12_trigger.c

```diff
@@ -128,7 +128,7 @@ static int lis2dw12_handle_drdy_int(struct device *dev)
 static int lis2dw12_handle_single_tap_int(struct device *dev)
 {
 	struct lis2dw12_data *data = dev->driver_data;
-	sensor_trigger_handler_t handler = data->tap_handler;;
+	sensor_trigger_handler_t handler = data->tap_handler;
```

Comment:
This changes are out of place here, please move it to a separate commit. Thanks.

Reply:
In a separate commit of the same pull request ?

Reply:
Yes

Reply:
Done !